### PR TITLE
[parallel-letter-frequency]: Test non-integer multiple of threads

### PR DESCRIPTION
**Pull request by ClashTheBunny**
999 % 4 != 0

### CODE REVIEW

File: exercises/parallel-letter-frequency/tests/parallel-letter-frequency.rs [hunk 1]
```diff
@@ -124,3 +124,18 @@ fn test_all_three_anthems_3_workers() {
...
     assert_eq!(freqs.get(&'ü'), Some(&2));
 }
+
+#[test]
+#[ignore]
+fn test_non_integer_multiple_of_threads() {
+    let mut v = Vec::with_capacity(999);
+    for _ in 0..999 {
+        v.push("abc");
+    }
```
Comment (coriolinus) [inline exercises/parallel-letter-frequency/tests/parallel-letter-frequency.rs:134]: I would expect clippy to request `.resize` or static creation here.

Comment (ClashTheBunny) [reply exercises/parallel-letter-frequency/tests/parallel-letter-frequency.rs:134]: Oops!  Trying to split up my three PRs didn't go as well!

**Comment by ClashTheBunny** [comment]
Okay, all tests pass with static creation.

**Comment by ClashTheBunny** [comment]
Oh fortuna! Pull 999 adding a 999 test!

**Review by coriolinus — APPROVED** [review]

**Pull request merged**